src/test/java/io/gravitee/notifier/webhook/AbstractWebhookNotifier.java

```diff
@@ -36,6 +36,18 @@ protected static BiFunction<Void, Throwable, Object> success(VertxTestContext te
         };
     }
 
+    protected static BiFunction<Void, Throwable, Object> error(VertxTestContext testContext) {
+        return (unused, throwable) -> {
+            if (throwable != null) {
+                testContext.completeNow();
+            } else {
+                testContext.failNow(new IllegalStateException("URL reachable"));
```

Comment:
Probably better to change it into "An error was expected but none has been thrown" wdyt?

Reply:
sure! I fp